firmware/controllers/algo/engine_configuration.cpp

```diff
@@ -765,9 +812,6 @@ void setDefaultConfiguration(DECLARE_ENGINE_PARAMETER_SIGNATURE) {
 // todo: this value is way off! I am pretty sure temp coeffs are off also
 	engineConfiguration->iat.config.bias_resistor = 2700;
 
-	engineConfiguration->auxTempSensor1.adcChannel = EFI_ADC_NONE;
```

Comment:
All "EFI_ADC_NONE" and "GPIO_UNASSIGNED" fields moved to prepareVoidConfiguration()